src/openforms/forms/api/serializers.py

```diff
@@ -220,13 +257,60 @@ def validate_backend_options(self, attrs, backend_field, options_field, registry
             detail = {options_field: e.detail}
             raise serializers.ValidationError(detail) from e
 
+    def validate_confirmation_email_template(self, validated_data):
+        if validated_data:
+            if validated_data.get("subject") and not validated_data.get("content"):
+                raise serializers.ValidationError(
+                    {
+                        "content": [
+                            ErrorDetail(
+                                string=_("This field is required."), code="required"
+                            )
+                        ]
+                    }
+                )
+
+            if validated_data.get("content") and not validated_data.get("subject"):
+                raise serializers.ValidationError(
+                    {
+                        "subject": [
+                            ErrorDetail(
+                                string=_("This field is required."), code="required"
+                            )
+                        ]
+                    }
+                )
+            for tag in [
+                "{% appointment_information %}",
+                "{% payment_information %}",
+            ]:
```

Comment:
We have `openforms.emails.validators.DjangoTemplateValidator` already for this functionality.